doc/tools.rst

```diff
@@ -0,0 +1,30 @@
+.. _tools:
+
+:octicon:`terminal` Tools
+=========================
+
+The python-toolbox ships with a set of command line tools, whose entry point always is the command :code:`tbx`.
+The commands are structured in a *tree* manner, and help is provided along with the command(s) no matter the nesting.
```

Comment:
As an author I usually find it helpful to prefer the singular, as it forces me to avoid ambiguity:

```suggestion
The commands are structured in a *tree* manner, and help is provided along with each command no matter the nesting.
```

Reply:
This section will be changed significantly.